monero-node/umbrel-app.yml

```diff
@@ -0,0 +1,22 @@
+manifestVersion: 1
+id: monero-node
+category: Finance
+name: Monero Node
+version: "0.18.2.2"
+tagline: A simple monero node runner
```

Comment:
Capitalize "Monero" here.

Reply:
now this is a huge mistake on my end. fixed!